minitest/shared/src/main/scala/hedgehog/minitest/HedgehogSupport.scala

```diff
@@ -0,0 +1,39 @@
+package hedgehog.minitest
+
+import minitest.SimpleTestSuite
+import hedgehog.runner.{SeedSource, Test}
+import hedgehog.core.{PropertyConfig, Seed, Status}
+import hedgehog.{Property, Result}
+
+trait HedgehogSupport { self: SimpleTestSuite =>
+  private val seedSource = SeedSource.fromEnvOrTime()
+  private val seed: Seed = Seed.fromLong(seedSource.seed)
+
+  def property(name: String, config: PropertyConfig = PropertyConfig.default)(
```

Comment:
`withConfig: PropertyConfig => PropertyConfig` would give even more power.

Reply:
Good idea, @dwijnand! I changed the `property` and `example` methods in `HedgehogSupport` accordingly.